packages/hardhat-core/src/internal/hardhat-network/provider/vm/exit.ts

```diff
@@ -47,7 +48,7 @@ export class Exit {
       default: {
         // TODO temporary, should be removed in production
```

Comment:
Should we remove the TODO?

Reply:
Addressed by https://github.com/NomicFoundation/hardhat/pull/4499/commits/f983666a5336bc78cef9dbd70a685d0410b2e81d